src/AsyncWebSocket.cpp

```diff
@@ -574,7 +574,7 @@ void AsyncWebSocketClient::_onData(void* pbuf, size_t plen) {
       break;
     }
 
-    // restore byte as _handleEvent may have added a null terminator i.e., data[len] = 0;
+    // restore byte as _handleEvent may have added a null terminator e.g., data[len] = 0;
```

Comment:
This one needs to be `i.e.` not `e.g.` 😅
i.e. => in other words
e.g. => for example

Reply:
thanks! always confusing the 2 😅